lib/board.rb

```diff
@@ -0,0 +1,46 @@
+class Board
+
+  attr_accessor :cells, :player
+
+  def initialize()
+    reset!
+  end
+
+  def reset!
+    @cells = []
+    9.times{@cells << " "}
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    cells[input.to_i - 1]
+  end
+
+  def update(occupy, player)
+     cells[occupy.to_i - 1] = player.token
+  end
+
+  def full?
+    true unless cells.any?{|x| x == " "}
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression. You can reverse (or negate) the boolean with the `!` (bang) operator.

```
def full?
  !cells.any?{|x| x == " "}
end
```

Of course, instead of using `!any?`, you could just use `none?` instead.